engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -193,8 +195,7 @@ namespace dmGameObject
         m_WorldTransforms.SetCapacity(max_instances);
         m_WorldTransforms.SetSize(max_instances);
         m_IDToInstance.SetCapacity(dmMath::Max(1U, max_instances/3), max_instances);
-        // TODO: Un-hard-code
-        m_InputFocusStack.SetCapacity(16);
+        m_InputFocusStack.SetCapacity(max_input_stack_entries);
```

Comment:
indentation seems off

Reply:
Huh? How? Where? Looks ok to me. Are you not using 4 spaces?

Reply:
Hmm, I think it's github that does something "clever" with line breaks when displaying the diff that makes it look confusing.